pkg/manager/rollbackconfig.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2019-present Open Networking Foundation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package manager
+
+import (
+	"fmt"
+	"github.com/onosproject/onos-config/pkg/store"
+	"github.com/onosproject/onos-config/pkg/store/change"
+	log "k8s.io/klog"
+)
+
+// RollbackTargetConfig rollbacks the last change for a given configuration on the target. Only the last one is
+// restored to the previous state. Going back n changes in time requires n sequential calls of this method.
+func (m *Manager) RollbackTargetConfig(configname string) error {
+	targetID := m.ConfigStore.Store[store.ConfigName(configname)].Device
+	log.Infof("Rolling back last change on config %s for target %s", configname, targetID)
+	updates, deletes, err := computeRollback(m, targetID, configname)
+	if err != nil {
+		return err
+	}
+	_, _, errSet := m.SetNetworkConfig(store.ConfigName(configname), updates, deletes)
+	//TODO if error we might want to take further action
+	return errSet
+}
+
+func computeRollback(m *Manager, target string, configname string) (map[string]string, []string, error) {
+	id, err := m.ConfigStore.RemoveLastChangeEntry(store.ConfigName(configname))
```

Comment:
I would like if the ID was returned at the end - just for confirmation (from client side) it would be good to know which change was rolled back

Reply:
done